specification/sql/resource-manager/Microsoft.Sql/preview/2021-05-01-preview/ManagedInstanceOperations.json

```diff
@@ -326,9 +326,20 @@
         }
       }
     },
-    "UpsertManagedServerOperationStep": {
+    "UpsertManagedServerOperationStepWithEstimatesAndDuration": {
       "type": "object",
       "properties": {
+        "stepStartTime": {
+          "format": "date-time",
+          "type": "string"
+        },
+        "stepEndTime": {
+          "format": "date-time",
+          "type": "string"
+        },
+        "timeElapsed": {
+          "type": "string"
```

Comment:
any particular format? Or description of one?

Reply:
timeElapsed is a string that returns the number of minutes passed from stepStartTime

Reply:
OK, could you add that as the description?

Reply:
added description in 6a3633278a825c2d19fc8edfc37650175638bb24